src/DataTable/TableControlBar.jsx

```diff
@@ -21,7 +29,8 @@ const TableControlBar = ({
           <div className="pgn__data-table-actions-left">
             <DropdownFilters />
           </div>
-          <div className="pgn__data-table-actions-right">
+          <div className={actionsSectionClassName}>
```

Comment:
To follow-up from our Slack huddle, we'll want to address the positioning of the view toggle component such that the view toggle component is vertically centered with the "Showing X of Y" and/or "1 selected, Select all 7, Clear selection" text when `togglePlacement="bottom"`.

We may want to move the view toggle component outside of the `div` intended for the table actions?

![image](https://user-images.githubusercontent.com/2828721/148260037-3e87891c-f0ee-4693-bed7-7eca84a81209.png)

Reply:
agreed, I think moving it out of actions makes sense too in general, plus it can help with the above layout fix